ghost/core/core/server/services/newsletters/NewslettersService.js

```
@@ -52,19 +52,22 @@ class NewslettersService {
                 return `Verify email address`;
             },
             getText(url, type, email) {
+                const expiration_message = 'For your security, the link will expire in 24 hours time.';
```

ESLint

[error] Identifier 'expiration_message' is not in camel case.

(camelcase)